ocaml/xapi/xapi_xenops.ml

```diff
@@ -259,6 +259,13 @@ let firmware_of_vm vm =
   | exception Not_found ->
       default_firmware
 
+let tpm_of_vm vm =
+  match List.assoc_opt "vtpm" vm.API.vM_platform with
```

Comment:
Should this use `String.lowercase_ascii` and also match `1`?

Reply:
I should try harded indeed, I'm surprised surrounding code doesn't use the platformdata code to process these fields, will change this here, then other code